test/e2e/main_test.go

```diff
@@ -0,0 +1,42 @@
+package e2e
+
+import (
+	"flag"
+	"log"
+	"os"
+	"testing"
+
+	"github.com/brancz/kube-rbac-proxy/test/kubetest"
+)
+
+// Sadly there's no way to pass Suite from TestMain to Test,
+// so we need this global instance
+var suite *kubetest.Suite
```

Comment:
You could also not have a `TestMain` function and do the entire setup inside `Test`. What do you think?

Reply:
Yes. I am thinking about extracting this package and then add some proper linting and code review on it. :+1:

Reply:
Sadly we need to use the `TestMain` to add flags like `-kubeconfig`. On the other hand we could also do these simply by using env vars?!